ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/ChatReply.qml

```diff
@@ -47,7 +47,7 @@ Loader {
                 visible: repliedMessageType != Constants.imageType
                 anchors.top: lblReplyAuthor.bottom
                 anchors.topMargin: 5
-                text: Emoji.parse(Utils.linkifyAndXSS(repliedMessageContent), "26x26");
+                text: Emoji.parse(Utils.linkifyAndXSS(repliedMessageContent), "72x72");
```

Comment:
Wouldn't the emojies look extremly big with 72x72?

Reply:
No, because the size parameter for `.parse()` specifies the source image size to retrieve from the filesystem, but the `<img/>` width and height attributes for the rendered emoji downscales the emoji but triples its pixel density (making it sharper on high density screens).